src/main/java/io/strimzi/kafka/bridge/SourceBridgeEndpoint.java

```diff
@@ -45,6 +50,10 @@
     private KafkaProducer<String, byte[]> producerUnsettledMode;
     private KafkaProducer<String, byte[]> producerSettledMode;
 
+    protected final String SEPARATOR = "#";
+    protected final int TOPIC_NOT_FOUND = 40401;
+    protected final int PARTITION_NOT_FOUND = 40402;
```

Comment:
These are more specific to HTTP, can we move them to the related HTTP source bridge endpoint? Wdyt?